src/backend/core/api/viewsets.py

```diff
@@ -214,10 +212,14 @@ def get_queryset(self):
                         Func("identities__email", function="unaccent"),
                         Func(Value(query), function="unaccent"),
                     )
+                    + TrigramSimilarity(
+                        Func("identities__name", function="unaccent"),
+                        Func(Value(query), function="unaccent"),
+                    )
```

Comment:
I might need to add a bit more tests, it feels a bit random when searching user. I have decided to follow contact's implementation, but I could opt for another approach